src/main/java/cms/gov/madie/measure/resources/TestCaseLockController.java

```diff
@@ -0,0 +1,34 @@
+package cms.gov.madie.measure.resources;
+
+import java.security.Principal;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import cms.gov.madie.measure.dto.LockInfo;
+import cms.gov.madie.measure.services.TestCaseLockService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+public class TestCaseLockController {
+  private final TestCaseLockService testCaseLockService;
+
+  @PostMapping("/measures/{measureId}/test-cases/{testCaseId}/testcase-lock")
```

Comment:
because test-cases is already in path maybe just `/lock` is fine? same for delete mapping
```suggestion
  @PostMapping("/measures/{measureId}/test-cases/{testCaseId}/lock")
```

Reply:
@adongare sure thanks!